### ISSUE TITLE
Investigate and fix test_check_site_health

### ISSUE THREAD

**Issue by gst** [savoirfairelinux/monitoring-tools#49]
which I had to disable in https://github.com/savoirfairelinux/monitoring-tools/commit/e0c9eedc3f1854e1549f92495ceb868cb6528b74  because the test fails.

See : https://travis-ci.org/savoirfairelinux/monitoring-tools/builds/63568422#L1362

**Comment by Seb-Solon** [comment]
beacause now it redirects to ring.cx :D

**Comment by gst** [comment]
yeah i saw that.. but something must be done for the test anyway ;)
